curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6401d7d41d61f94ca4211b7f.md

```diff
@@ -0,0 +1,175 @@
+---
+id: 6401d7d41d61f94ca4211b7f
+title: Step 22
+challengeType: 0
+dashedName: step-22
+---
+
+# --description--
+
+Now, you need to add a click event listener to the "Load More Authors" button. In step 1, you selected the button and put it inside the variable `loadMoreBtn`.
+
+To add an event listener in JavaScript, you do it this way:
+
+```js
+element.addEventListener(eventType, functionToRun)
+```
+
+The `eventType` could be `hover`, `keyup`, `keydown`, `click`, and many more. The `functionToRun` is the function that runs once the event fires.
```

Comment:
While this is a good explanation, I don't think we need it here since campers will have plenty of practice using event listeners in earlier projects

Reply:
Agreed. Learners should be familiar with both of these event listeners by the time they get to this project.